src/data-sources/intermine/api/search-traits.ts

```diff
@@ -17,22 +18,50 @@ import { PaginationOptions } from './pagination.js';
 
 export type SearchTraitsOptions = {
     name?: string;
+    studyType?: string;
+    publicationId: string;
```

Comment:
`publicationId` should be optional like the other options.

Reply:
Good eye! Fixed.